docs/guide/features.md

```diff
@@ -126,12 +126,15 @@ Alternatively, you can add `vite/client` to `compilerOptions.types` inside `tsco
 ```json
 {
   "compilerOptions": {
-    "types": ["vite/client"]
+    "types": ["vite/client", "some-other-global-lib"]
   }
 }
 ```
 
-This will provide the following type shims:
+::: warning Note that [`compilerOptions.types`](https://www.typescriptlang.org/tsconfig#types) will override any existing global types.
```

Comment:
I think the wording in the TS docs is more clear:
```suggestion
::: warning If [`compilerOptions.types`](https://www.typescriptlang.org/tsconfig#types) is specified, only these packages will be included in the global scope (instead of all visible ”@types” packages).
```